<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportTab</class>
 <widget class="QWidget" name="ImportTab">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing"><number>20</number></property>

   <item>
    <widget class="QLabel" name="label_title">
     <property name="text"><string>Import Data</string></property>
     <property name="font"><font><pointSize>16</pointSize><bold>true</bold></font></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_type">
     <property name="title"><string>1. Select Data Type</string></property>
     <layout class="QHBoxLayout" name="horizontalLayout_type">
      <item>
       <widget class="QRadioButton" name="radio_menu_items">
        <property name="text"><string>Menu Items</string></property>
        <property name="checked"><bool>true</bool></property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_employees">
        <property name="text"><string>Employees</string></property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QGroupBox" name="groupBox_file">
     <property name="title"><string>2. Select JSON File</string></property>
     <layout class="QHBoxLayout" name="horizontalLayout_file">
      <item>
       <widget class="QLineEdit" name="lineEdit_file_path">
        <property name="placeholderText"><string>No file selected...</string></property>
        <property name="readOnly"><bool>true</bool></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_browse">
        <property name="text"><string>Browse...</string></property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QPushButton" name="btn_import">
     <property name="text"><string>Import Data</string></property>
     <property name="minimumHeight"><number>50</number></property>
     <property name="font"><font><pointSize>12</pointSize><bold>true</bold></font></property>
    </widget>
   </item>

   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>